<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Details</string>
  </property>
  <widget class="QPushButton" name="Ensure">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密钥key：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="key1_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>93</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>a:</string>
   </property>
  </widget>
  <widget class="QLabel" name="key2_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>133</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>b:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Encrypt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>98</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Decrypt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>98</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Playfair_key">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="From">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="To">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Tips_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <zorder>Tips_text</zorder>
  <zorder>Playfair_key</zorder>
  <zorder>Ensure</zorder>
  <zorder>Cancel</zorder>
  <zorder>label</zorder>
  <zorder>spinBox</zorder>
  <zorder>spinBox_2</zorder>
  <zorder>key1_name</zorder>
  <zorder>key2_name</zorder>
  <zorder>Encrypt</zorder>
  <zorder>Decrypt</zorder>
  <zorder>From</zorder>
  <zorder>To</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
